serverless/installing-kn.adoc

```diff
@@ -18,5 +18,7 @@ For more information on installing the `oc` CLI for your operating system and lo
 include::modules/serverless-installing-cli-web-console.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-linux-rpm.adoc[leveloffset=+1]
 include::modules/serverless-installing-cli-linux.adoc[leveloffset=+1]
-include::modules/serverless-installing-cli-macos.adoc[leveloffset=+1]
+include::serverless-installing-cli-linux-ibm-power.adoc[leveloffset=+1]
+include::serverless-installing-cli-linux-ibm-z.adoc[leveloffset=+2]
+include::modules/serverless-installing-cli-macos.adoc[leveloffset=+2]
```

Comment:
Because these are not created as modules properly, nothing is showing up here for https://deploy-preview-29918--osdocs.netlify.app/openshift-enterprise/latest/serverless/installing-kn.html